app/controllers/users.js

```diff
@@ -0,0 +1,38 @@
+const users = require('../models').users,
+  bcrypt = require('bcryptjs'),
+  helpers = require('../helpers'),
+  logger = require('../logger'),
+  errors = require('../errors');
+
+exports.signUp = (req, res, next) => {
+  const saltRounds = 5;
+  const signErrors = [];
+
+  if (!helpers.validatePassword(req.body.password)) signErrors.push(errors.invalidPassword());
```

Comment:
theres something weird here. You should push messages and throw a generic error. and dont throw in the helpers validations, just return the message